recipe/run_test.py

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env python
 
 import os
+import shutil
 
 
+# move lena.png and boat.png out
+for file in ('extdata/lena.png', 'extdata/boat.png'):
+    if os.path.exists(file):
+        shutil.move(file, '.')
```

Comment:
As `file` is a builtin can we use something else (e.g. `fn`)?

Reply:
Sorry never know this (I thought only `open` is built in). Will fix it.